lib/hydra/works/services/determine_mime_type.rb

```diff
@@ -0,0 +1,24 @@
+module Hydra::Works
+  class DetermineMimeType
+    # Determines the mime type for a given file
```

Comment:
I believe this method documentation should sit above the `#call` method definition rather than above `class << self`.